### ISSUE TITLE
Using vim as the git message editor causes chaos

### ISSUE THREAD

**Issue by doctorpangloss** [rmyorston/busybox-w32#324]
This is probably an interaction with the `vim` binary?

![ConEmu64_B0Y9mMiGoZ](https://github.com/rmyorston/busybox-w32/assets/2229300/365b123b-b3c1-4ae5-9e76-3cb98ef1e85f)

`git rebase -i HEAD~1` in a git repository.

**Comment by rmyorston** [comment]
I don't see any problem: `vim` works just as I'd expect when invoked via `git rebase` in ConEmu.

**Comment by doctorpangloss** [comment]
Hmm, this is the case on every machine I test. Let me investigate further.

**Comment by rmyorston** [comment]
My test was with Windows 10, ConEmu 221218 and Git 2.39.2.

A fresh installation on Windows 11 with Git 2.40.1 did exhibit the problem.

I upgraded the Windows 10 installation to Git 2.40.1 too, but it still worked properly.

I also tried a fresh installation of Git 2.39.2 on Windows 8, which worked correctly and continued to do so after an upgrade to Git 2.40.1.

However, I note that the problem is still present on Windows 11 when Git is run from ConEmu + cmd or ConEmu + PowerShell. I conclude that busybox-w32 isn't the root cause.
